test/functional/p2p_ibd.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test inv messages and fee filters during and after IBD."""
+
+from decimal import Decimal
+
+from test_framework.messages import COIN
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+MAX_FEE_FILTER = Decimal(str(9170997 / COIN))
+NORMAL_FEE_FILTER = Decimal(str(100 / COIN))
+
+
+class P2PIBDTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+        self.extra_args = [["-minrelaytxfee=0.00000100"], ["-minrelaytxfee=0.00000100"]]
```

Comment:
Could use the constant?
```suggestion
        self.extra_args = [
        ["-minrelaytxfee={}".format(NORMAL_FEE_FILTER)],
        ["-minrelaytxfee={}".fromat(NORMAL_FEE_FILTER)],
         ]
```